### ISSUE TITLE
Server side validation

### ISSUE THREAD

**Issue by nigelmpofu** [teampinocchio/pinocchio#150]
In most cases validation is only done on the client side so that leaves the server vulnerable to any changes in the client side html code.

**Comment by devosray** [comment]
Good point. Maybe we can add a checkbox list to this issue listing all the places that need server-side validation?

**Comment by nigelmpofu** [comment]
### Server side validation

- [ ] User management (Creating/Editing users - is formatting correct)
- [ ] Create/Maintain rounds (Are the dates still dates)
- [ ] Question and questionnaire admin (Are required fields not blank)
- [ ] Questionnaire responses (Are the responses within an acceptable range, rating not exceeding limits)
